cocos/core/data/utils/attribute-internal.ts

```diff
@@ -0,0 +1,12 @@
+import { Enum, EnumType } from '../../value-types/enum';
+import { getClassAttrs, DELIMETER } from './attribute';
+
+// eslint-disable-next-line @typescript-eslint/ban-types
```

Comment:
Why need it?

Reply:
![image](https://github.com/cocos/cocos-engine/assets/28807867/e65273bb-3220-4785-97c7-d867565767ab)